package.json

```diff
@@ -12,10 +12,13 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "eslint": "2.9.x",
+    "eslint": "3.19.x",
     "ramda": "0.21.x",
     "xyz": "2.0.x"
   },
+  "peerDependencies": {
+    "eslint": "3.19.x"
+  },
```

Comment:
This ensures that projects which depend on `sanctuary-style` must also depend on the expected version of `eslint`.